### ISSUE TITLE
No toolbar on Ubuntu 15.10

### ISSUE THREAD

**Issue by romu70** [kimchi-project/kimchi#846]
Hi,
I finally managed to install Kimchi on a Ubuntu 15.10. I can reach the https://localhost:8001 page and I can login. Now, I've another annoying issue : there is no toolbar. I see the title bar with the Kimchi icon, the user menu with a "Log out" item and the help menu.

Below, there is an empty grey area where I guess the toolbar should stand...but it doesn't.

Any help would be great. Thanks.

**Comment by romu70** [comment]
More on this issue.

I've tried to remove Kimchi by running ```sudo make uninstall```, so while keeping only Wok and GingerBase. And surprise! I see the "Host" top menu with the "Dashboard" and "Updates" menu items.

So now, how can I have a properly running Kimchi?

**Comment by romu70** [comment]
Another update, I tried to install Kimchi 2.0.0 instead of the master. Problem, Kimchi with the tag 2.0.0 doesn't build. Problem comes from sassc command doesn't exist from scratch on Ubuntu.

So, I built sassc. But this doesn't work because the commande ```sassc -s expanded src/kimchi.scss kimchi.css``` is not correct, and should be (accord the sassc help): ```sassc -t expanded src/kimchi.scss kimchi.css```. But I didn't find where to fix this in the makefile.

I'm feeling a bit lonely on this thread, any help?
